src/styles/blocks/categories.scss

```diff
@@ -0,0 +1,104 @@
+.categories {
+  margin-bottom: 96px;
+
+  @include on-tablet {
+    margin-bottom: 120px;
+  }
+
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+    margin-bottom: 40px;
+
+    @include on-tablet {
+      row-gap: 56px;
+    }
+  }
+}
+
+.category {
+  &__photos {
+    @include page-grid;
+
+    margin-bottom: 16px;
+  }
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #f9f9f9;
+    grid-column: span 1;
+
+    @include hover(transform, scale(1.2));
```

Comment:
the scale is too large.
Change it to `1.1` so that elements don't overlap
```suggestion
    @include hover(transform, scale(1.1));
```
<img width="1268" alt="Знімок екрана 2024-10-22 о 14 27 22" src="https://github.com/user-attachments/assets/3f62c085-4503-429b-ab86-4087e3b8eed2">